lib/fluent/plugin/filter_kubernetes_metadata.rb

```diff
@@ -366,32 +364,22 @@ def start_watch
       watcher.each do |notice|
         case notice.type
           when 'MODIFIED'
-            if notice.object.status.containerStatuses
```

Comment:
TBH it probably wasn't filtering much before - containerStatuses are sent with the object anyway. I guess it could filter out pods that are created but have no containers running. OK will add this back in.